Obj-C/AEPSampleAppObjC/AppDelegate.m

```diff
@@ -14,7 +14,7 @@ @implementation AppDelegate
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
     
     [AEPMobileCore setLogLevel: AEPLogLevelTrace];
-    NSArray *extensionsToRegister = @[AEPMobileIdentity.class, AEPMobileLifecycle.class, AEPMobileSignal.class];
+    NSArray *extensionsToRegister = @[AEPMobileIdentity.class, AEPMobileLifecycle.class, AEPMobileSignal.class, AEPMobileMessaging.class];
```

Comment:
Are we not adding `setPushIdentifier` API to objectiveC app ?

Reply:
I did not update the objective c app